### ISSUE TITLE
[Bug]: Missing sandbox_high setting

### ISSUE THREAD

**Issue by SGZ-Creations** [refringe/CustomRaidTimes#15]
### Discord Username

sgamez

### What happened?

The issue is that after level 20 the ground zero map (sandbox_high) revert the timer to default and all my setting are adjusted while this issues is not happening anywhere else 3.9.8.
[config-copy.txt](https://github.com/user-attachments/files/16784008/config-copy.txt)
Made it txt purely to post it on git.

### SPT Version

3.9.1

### Server Log

```shell
Commit hash: a0c79ac6bc0e69663b5a749c9f8d515a74a8ccfa
Build date: 1724054904726
┌─────────────────────────────────────────┐
│ SPT 3.9.8                               │
│ https://discord.sp-tarkov.com           │
│                                         │
│ This work is free of charge             │
│ If you paid money, you were scammed     │
│ Commercial use is prohibited            │
└─────────────────────────────────────────┘
ModLoader: loading: 47 server mods...
Mod: WTTArmoryM107 version: 1.0.4 by: Rexana loaded
Mod: Artem Trader version: 1.8.3 by: Crackbone loaded
Mod: Artem Equipment version: 1.8.2 by: Crackbone loaded
Mod: Viper Items version: 1.0.0 by: RheddElBozo loaded
Mod: Bosses Have Lega Medals version: 1.0.0 by: acidphantasm loaded
Mod: RefSPTFriendlyQuests version: 1.1.0 by: acidphantasm loaded
Mod: Scorpion version: 0.11.3 by: acidphantasm loaded
Mod: Adam&#39;s Boxes at Ref (BARF) version: 0.2.5 by: Adam loaded
Mod: BRNVG - N-15 Adapter version: 1.5.7 by: Borkel loaded
Mod: TraderModding version: 1.8.0 by: ChooChoo loaded
Mod: WeightTweaker version: 1.4.0 by: CreamCheese loaded
Mod: LiveFleaPrices version: 1.2.0 by: DrakiaXYZ loaded
Mod: Expanded Task Text version: 1.5.3 by: Dirtbikercj loaded
Mod: Fontaine FOV Fix &amp; Variable Optics version: 2.1.1 by: Fontaine loaded
Mod: give-ui version: 1.4.0 by: agavalda loaded
Mod: AchievementsPercentageLive version: 1.1.0 by: GoldenHeartbell loaded
Mod: configurable-inventories version: 1.5.0-beta by: Harmer loaded
Mod: IgorTrader version: 1.0.6 by: Pycak loaded
Mod: Gilded Key Storage version: 1.4.0 by: Jehree loaded
Mod: removetediousquestconditions version: 2.5.0 by: Kiki loaded
Mod: progressive_container version: 1.2.2 by: leaves loaded
Mod: Lotus version: 1.3.7 by: Lunnayaluna loaded
Mod: AutoProfileBackup version: 1.5.0 by: Lua loaded
Mod: MAPE version: 1.2.0 by: 3xtremehamster loaded
Mod: MedicalAttention version: 395.0.2 by: jbs4bmx loaded
Mod: MoreCheckmarksBackend version: 1.5.14 by: VIP loaded
Mod: MusicManiac-LessRestrictingHeadwear version: 2.1.0 by: MusicManiac loaded
Mod: Realistic Bot Names version: 0.0.1 by: PenOkOh loaded
Mod: Props Automated Selling version: 1.6.0 by: Props loaded
Mod: AmmoStats version: 3.1.2 by: RaiRaiTheRaichu loaded
Mod: Realistic-Thermals version: 1.0.4 by: Borkel loaded
Mod: Increase Climb Height version: 1.2.0 by: redlaser42 loaded
Mod: custom-camera-recoil version: 1.2.0 by: Refringe loaded
Mod: custom-raid-times version: 1.7.2 by: Refringe loaded
Mod: S.T.A.L.K.E.R 2 (2011) Duty Exoskeleton version: 5.0.1 by: SamSWAT loaded
Mod: SVM version: 1.9.1 by: GhostFenixx loaded
Mod: Skills Extended version: 1.3.0 by: dirtbikercj loaded
Mod: SpecialSlots version: 390.0.2 by: jbs4bmx loaded
Mod: TheMercenary version: 3.1.0 by: FurryFailure loaded
Mod: uifixes version: 2.4.2 by: Tyfon loaded
Mod: Viper version: 1.0.0 by: RheddElBozo loaded
Mod: Virtuals Custom Quest Loader version: 2.0.2 by: Virtual loaded
Mod: WTT-LittleDrummerBoy version: 1.0.5 by: Tron loaded
Mod: Zero2Hero++ version: 1.1.1 by: TheDevilsrevenge loaded
Mod: CustomLauncherBackgrounds version: 1.0.1 by: PreyToLive loaded
Mod: CustomTraderAvatars version: 1.0.1 by: PreyToLive loaded
Mod: AllTheLoot version: 1.0.8 by: lBlackMambal loaded
[WTTArmoryM107] ------------------------------------------------------------------------
[WTTArmoryM107] Pre-Alpha development build
[WTTArmoryM107] Developers:           Rexana & Tron
[WTTArmoryM107] You can kill it. We can help.
[WTTArmoryM107] ------------------------------------------------------------------------
[AAAViperItems] You're In For The LongHaul Kid
Skills Extended loading
[TheMercenary 3.1.0]// !CONNECTING... //: TARKOV -> LIMANSK //
[TheMercenary 3.1.0]// !CONNECTION RECIEVED! //: TRANSFERRING HIDEOUT LOCALE //
[TheMercenary 3.1.0]// !INCOMING! //: LIMANSK -> TARKOV // Mercenary convoy enroute. //
Server: executing startup callbacks...
Importing database...
Database import finished.
[WTTArmoryM107] Database: Loaded 20 custom items.
[WTTArmoryM107] Database: Loading complete.
SVM 1.9.1 has initialized, Sanitar did nothing wrong
SVM Preset - 3.9.8 Version 6.0.4 - successfully loaded
SVM: Custom Properties enabled
[ '#T H I C C item case' ]
[ '5c0a840b86f7742ffa4f2482', 'Weight', '0' ]
[ '' ]
[ '#T H I C C Weapon case' ]
[ '5b6d9ce188a4501afc1b2b25', 'Weight', '0' ]
[ '' ]
[ '#Repair Kits' ]
[ '5910968f86f77425cf569c32', 'MaxRepairResource', '50000' ]
[ '591094e086f7747caa7bb2ef', 'MaxRepairResource', '1000000' ]
[ '' ]
[ '#Grizzly medical kit' ]
[ '590c657e86f77412b013051d', 'MaxHpResource', '10000' ]
[ '' ]
[ '#Raptar' ]
[ '61605d88ffa6e502ac5e7eeb', 'Ergonomics', '95' ]
[ '61605d88ffa6e502ac5e7eeb', 'Accuracy', '100' ]
[ '61605d88ffa6e502ac5e7eeb', 'Recoil', '-100' ]
[ '' ]
[ '#Zeus-Pro-Thermal' ]
[ '63fc44e2429a8a166c7f61e6', 'Ergonomics', '95' ]
[ '63fc44e2429a8a166c7f61e6', 'Accuracy', '100' ]
[ '63fc44e2429a8a166c7f61e6', 'Recoil', '-100' ]
[ '' ]
[ '#DVL-10 7.62x51 10-round magazine' ]
[ '5888988e24597752fe43a6fa', 'Cartridges', '0', '_max_count', '100' ]
[ '' ]
[ '#Kalashnikov PKM 7.62x54R machine gun' ]
[ '64637076203536ad5600c990', 'Durability', '10000' ]
[ '64637076203536ad5600c990', 'MaxDurability', '10000' ]
[ '' ]
[ '#PK 7.62x54R 100-round box' ]
[ '646372518610c40fc20204e8', 'Cartridges', '0', '_max_count', '500' ]
[ '' ]
[ '#.338 mags Mk18, AXMC' ]
[ '5fc23426900b1d5091531e15', 'Cartridges', '0', '_max_count', '100' ]
[ '628120fd5631d45211793c9f', 'Cartridges', '0', '_max_count', '100' ]
[ '' ]
[ '#Aug mag 42' ]
[ '630e295c984633f1fb0e7c30', 'Cartridges', '0', '_max_count', '75' ]
[ '' ]
[ '#mag_ar10_lancer_l7_awm_762x51_20' ]
[ '65293c38fc460e50a509cb25', 'Cartridges', '0', '_max_count', '50' ]
[ '65293c38fc460e50a509cb25', 'Ergonomics', '30' ]
[ '65293c38fc460e50a509cb25', 'Accuracy', '-15' ]
[ '65293c38fc460e50a509cb25', 'Recoil', '-20' ]
[ '' ]
[ '#mag_ar10_lancer_l7_awm_762x51_25' ]
[ '65293c7a17e14363030ad308', 'Cartridges', '0', '_max_count', '100' ]
[ '65293c7a17e14363030ad308', 'Ergonomics', '60' ]
[ '65293c7a17e14363030ad308', 'Accuracy', '-30' ]
[ '65293c7a17e14363030ad308', 'Recoil', '-40' ]
[ '' ]
[ '#6Sh118 raid backpack' ]
[ '5df8a4d786f77412672a1e3b', 'Weight', '0.5' ]
[ '' ]
[ '#Mystery Ranch NICE COMM 3 BVS frame system' ]
[ '628bc7fb408e2b2e9c0801b1', 'Weight', '1' ]
[ '' ]
[ '#Cult Termite ballistic plate' ]
[ '656fa99800d62bcd2e024088', 'Weight', '0' ]
[ '656fa99800d62bcd2e024088', 'armorClass', '10' ]
[ '656fa99800d62bcd2e024088', 'Durability', '25000' ]
[ '656fa99800d62bcd2e024088', 'MaxDurability', '25000' ]
[ '' ]
[ '#GAC 4sss2 ballistic plate' ]
[ '656faf0ca0dce000a2020f77', 'Weight', '0' ]
[ '656faf0ca0dce000a2020f77', 'armorClass', '10' ]
[ '656faf0ca0dce000a2020f77', 'Durability', '25000' ]
[ '656faf0ca0dce000a2020f77', 'MaxDurability', '25000' ]
[ '' ]
[ '#ESBI level IV ballistic plate (Side)' ]
[ '64afdb577bb3bfe8fe03fd1d', 'Weight', '0' ]
[ '64afdb577bb3bfe8fe03fd1d', 'armorClass', '10' ]
[ '64afdb577bb3bfe8fe03fd1d', 'Durability', '25000' ]
[ '64afdb577bb3bfe8fe03fd1d', 'MaxDurability', '25000' ]
[ '' ]
[ '#KITECO SC-IV SA ballistic plate' ]
[ '656fafe3498d1b7e3e071da4', 'Weight', '0' ]
[ '656fafe3498d1b7e3e071da4', 'armorClass', '10' ]
[ '656fafe3498d1b7e3e071da4', 'Durability', '25000' ]
[ '656fafe3498d1b7e3e071da4', 'MaxDurability', '25000' ]
[ '' ]
[ '#Granit Br5 ballistic plate' ]
[ '64afc71497cf3a403c01ff38', 'Weight', '0' ]
[ '64afc71497cf3a403c01ff38', 'armorClass', '10' ]
[ '64afc71497cf3a403c01ff38', 'Durability', '25000' ]
[ '64afc71497cf3a403c01ff38', 'MaxDurability', '25000' ]
[ '' ]
[ '#Granit ballistic plate (Side) - 6class' ]
[ '64afd81707e2cf40e903a316', 'Weight', '0' ]
[ '64afd81707e2cf40e903a316', 'armorClass', '10' ]
[ '64afd81707e2cf40e903a316', 'Durability', '25000' ]
[ '64afd81707e2cf40e903a316', 'MaxDurability', '25000' ]
[ '' ]
[ '#Granit 4RS ballistic plates (Front)' ]
[ '656f66b5c6baea13cd07e108', 'Weight', '0' ]
[ '656f66b5c6baea13cd07e108', 'armorClass', '10' ]
[ '656f66b5c6baea13cd07e108', 'Durability', '25000' ]
[ '656f66b5c6baea13cd07e108', 'MaxDurability', '25000' ]
[ '' ]
[ '#Granit 4RS ballistic plates (Back)' ]
[ '654a4a964b446df1ad03f192', 'Weight', '0' ]
[ '654a4a964b446df1ad03f192', 'armorClass', '10' ]
[ '654a4a964b446df1ad03f192', 'Durability', '25000' ]
[ '654a4a964b446df1ad03f192', 'MaxDurability', '25000' ]
[ '' ]
[ '#Korund-VM-K ballistic plates (Front)' ]
[ '656f63c027aed95beb08f62c', 'Weight', '0' ]
[ '656f63c027aed95beb08f62c', 'armorClass', '10' ]
[ '656f63c027aed95beb08f62c', 'Durability', '25000' ]
[ '656f63c027aed95beb08f62c', 'MaxDurability', '25000' ]
[ '' ]
[ '#Korund-VM-K ballistic plate (Back)' ]
[ '656fa61e94b480b8a500c0e8', 'Weight', '0' ]
[ '657b28d25f444d6dff0c6c77', 'armorClass', '10' ]
[ '657b28d25f444d6dff0c6c77', 'Durability', '25000' ]
[ '657b28d25f444d6dff0c6c77', 'MaxDurability', '25000' ]
[ '' ]
[ '#NESCO 4400-SA-MC ballistic plate' ]
[ '656fa61e94b480b8a500c0e8', 'Weight', '0' ]
[ '656fa61e94b480b8a500c0e8', 'armorClass', '10' ]
[ '656fa61e94b480b8a500c0e8', 'Durability', '25000' ]
[ '656fa61e94b480b8a500c0e8', 'MaxDurability', '25000' ]
[ '' ]
[ '#Kiba Arms Steel ballistic plate' ]
[ '656fa76500d62bcd2e024080', 'Weight', '0' ]
[ '656fa76500d62bcd2e024080', 'armorClass', '10' ]
[ '656fa76500d62bcd2e024080', 'Durability', '25000' ]
[ '656fa76500d62bcd2e024080', 'MaxDurability', '25000' ]
[ '' ]
[ '#Diamond Age NeoSteel High Cut helmet (Black) - Nape-Back' ]
[ '657f9ef6c6679fefb3051e1f', 'Durability', '2500' ]
[ '657f9ef6c6679fefb3051e1f', 'MaxDurability', '2500' ]
[ '657f9ef6c6679fefb3051e1f', 'armorClass', '10' ]
[ '' ]
[ '#Diamond Age NeoSteel High Cut helmet (Black) - Top' ]
[ '657f9eb7e9433140ad0baf86', 'Durability', '2500' ]
[ '657f9eb7e9433140ad0baf86', 'MaxDurability', '2500' ]
[ '657f9eb7e9433140ad0baf86', 'armorClass', '10' ]
[ '' ]
[ '#Diamond Age NeoSteel helmet ballistic mandible' ]
[ '6570a88c8f221f3b210353b7', 'Durability', '5000' ]
[ '6570a88c8f221f3b210353b7', 'MaxDurability', '5000' ]
[ '6570a88c8f221f3b210353b7', 'armorClass', '10' ]
[ '' ]
[ '#Ops-Core FAST Gunsight Mandible' ]
[ '5a16ba61fcdbcb098008728a', 'armorClass', '10' ]
[ '5a16ba61fcdbcb098008728a', 'Durability', '500' ]
[ '5a16ba61fcdbcb098008728a', 'MaxDurability', '500' ]
[ '' ]
[ '#NPP KlASS Tor-2 helmet face shield - Faceshield' ]
[ '65719f9ef392ad76c50a2ec8', 'armorClass', '10' ]
[ '65719f9ef392ad76c50a2ec8', 'Durability', '1000' ]
[ '65719f9ef392ad76c50a2ec8', 'MaxDurability', '1000' ]
[ '' ]
[ '#NPP KlASS Tor-2 helmet (Olive Drab) Ears' ]
[ '657fa186d4caf976440afe42', 'armorClass', '10' ]
[ '657fa186d4caf976440afe42', 'Durability', '500' ]
[ '657fa186d4caf976440afe42', 'MaxDurability', '500' ]
[ '' ]
[ '#NPP KlASS Tor-2 helmet (Olive Drab) Nape Back' ]
[ '657fa168e9433140ad0baf8e', 'armorClass', '10' ]
[ '657fa168e9433140ad0baf8e', 'Durability', '500' ]
[ '657fa168e9433140ad0baf8e', 'MaxDurability', '500' ]
[ '' ]
[ '#NPP KlASS Tor-2 helmet (Olive Drab) Top' ]
[ '657fa0fcd4caf976440afe3e', 'armorClass', '10' ]
[ '657fa0fcd4caf976440afe3e', 'Durability', '500' ]
[ '657fa0fcd4caf976440afe3e', 'MaxDurability', '500' ]
[ '' ]
[ '#Team Wendy EXFIL Ballistic Helmet (Black) Top' ]
[ '6551fec55d0cf82e51014288', 'armorClass', '10' ]
[ '6551fec55d0cf82e51014288', 'Durability', '1500' ]
[ '6551fec55d0cf82e51014288', 'MaxDurability', '1500' ]
[ '' ]
[ '#Team Wendy EXFIL Ballistic Helmet (Black) Nape Back' ]
[ '655200ba0ef76cf7be09d528', 'armorClass', '10' ]
[ '655200ba0ef76cf7be09d528', 'Durability', '1500' ]
[ '655200ba0ef76cf7be09d528', 'MaxDurability', '1500' ]
[ '' ]
[ '#Team Wendy EXFIL Ballistic face shield (Black)' ]
[ '5e00cdd986f7747473332240', 'armorClass', '10' ]
[ '5e00cdd986f7747473332240', 'Durability', '2000' ]
[ '5e00cdd986f7747473332240', 'MaxDurability', '2000' ]
[ '' ]
[ '#Team Wendy EXFIL Ear Covers (Black)' ]
[ '5e00cfa786f77469dc6e5685', 'armorClass', '10' ]
[ '5e00cfa786f77469dc6e5685', 'Durability', '2000' ]
[ '5e00cfa786f77469dc6e5685', 'MaxDurability', '2000' ]
[ '' ]
[ '#Team Wendy EXFIL Ballistic face shield (Coyote Brown)' ]
[ '5e01f37686f774773c6f6c15', 'armorClass', '10' ]
[ '5e01f37686f774773c6f6c15', 'Durability', '2000' ]
[ '5e01f37686f774773c6f6c15', 'MaxDurability', '2000' ]
[ '' ]
[ '#Team Wendy EXFIL Ear Covers (Coyote Brown)' ]
[ '5e01f31d86f77465cf261343', 'armorClass', '10' ]
[ '5e01f31d86f77465cf261343', 'Durability', '2000' ]
[ '5e01f31d86f77465cf261343', 'MaxDurability', '2000' ]
[ '' ]
[ '#Galvion Caiman Hybrid helmet (Grey) Top' ]
[ '657bbad7a1c61ee0c3036323', 'armorClass', '10' ]
[ '657bbad7a1c61ee0c3036323', 'Durability', '1500' ]
[ '657bbad7a1c61ee0c3036323', 'MaxDurability', '1500' ]
[ '' ]
[ '#Galvion Caiman Hybrid helmet (Grey) Nape Back' ]
[ '657bbb31b30eca9763051183', 'armorClass', '10' ]
[ '657bbb31b30eca9763051183', 'Durability', '1500' ]
[ '657bbb31b30eca9763051183', 'MaxDurability', '1500' ]
[ '' ]
[ '#Galvion Caiman Fixed Arm Visor' ]
[ '5f60bf4558eff926626a60f2', 'armorClass', '10' ]
[ '5f60bf4558eff926626a60f2', 'Durability', '2000' ]
[ '5f60bf4558eff926626a60f2', 'MaxDurability', '2000' ]
[ '' ]
[ '#Galvion Caiman Hybrid Ballistic Applique' ]
[ '5f60b85bbdb8e27dee3dc985', 'armorClass', '10' ]
[ '5f60b85bbdb8e27dee3dc985', 'Durability', '2500' ]
[ '5f60b85bbdb8e27dee3dc985', 'MaxDurability', '2500' ]
[ '' ]
[ '#Galvion Caiman Hybrid Ballistic Mandible Guard' ]
[ '5f60c076f2bcbb675b00dac2', 'armorClass', '10' ]
[ '5f60c076f2bcbb675b00dac2', 'Durability', '2500' ]
[ '5f60c076f2bcbb675b00dac2', 'MaxDurability', '2500' ]
SVM: Custom properties successfully loaded
Loading: Artem Equipment
[AAAViperItems] Database: Loaded 1 custom items.
[AAAViperItems] Database: Loading complete.
[VCQL] Loaded 363 custom quests.
[VCQL] 13 custom quests were pruned due to date settings.
[VCQL] Loaded 192 custom quest images.
[VCQL] Loaded 0 custom trader images.
[acidphantasm-scorpion] ┬─┬ノ( º _ ºノ)
Boxes Added to Ref
[SBNV]: Your N-15s can also be mounted on helmets now!
Flea Prices Updated!
Expanded Task Text is loading please wait...
Valid cache found. Merging saved tasks.
Expanded Task Text startup took 0.07971659999999975 seconds...
> Achievements Updated.
[Gilded Key Storage] : Mod loading
[Gilded Key Storage] : Secure Key Box loaded! Hooray!
[Gilded Key Storage] : Golden Keychain Mk. I loaded! Hooray!
[Gilded Key Storage] : Golden Keychain Mk. II loaded! Hooray!
[Gilded Key Storage] : Golden Keychain Mk. III loaded! Hooray!
[Gilded Key Storage] : Golden Keycard Holder loaded! Hooray!
[ProgressiveContainer] ProgressiveContainer Starting:
[ProgressiveContainer]  Adding craft for: pouch
[ProgressiveContainer]  Adding craft for: alpha
[ProgressiveContainer]  Adding craft for: beta
[ProgressiveContainer]  Adding craft for: epsilon
[ProgressiveContainer]  Adding craft for: gamma
[ProgressiveContainer]  Adding craft for: kappa
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
                                                Lotus arrived in Tarkov
<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

Loading: Lua-AutoProfileBackup 1.5.0
[MAPE] Artem Equipment is detected.
[MAPE] Make Armor Plate Effective Loaded
jbs4bmx-MedicalAttention v395.0.2: Cached Successfully
[LessRestrictingHeadwear] MusicManiac - Less Restricting Headwear Loaded
Importing BEAR names from the directory...
Importing USEC names from the directory...
All names have been successfully loaded without issues!
[AmmoStats]: Loaded successfully.
[AmmoStats]: ColorConverter Plugin loaded! Extended functionality enabled.
CustomCameraRecoil: Adjusted the camera recoil for 151 weapons.
CustomRaidTimes: Raid times have been successfully adjusted according to the configuration.
Skills Extended: Loaded 44 achievements
Skills Extended: Loaded 10 quests
Skills Extended: Loaded 7 custom items
Skills Extended: Loaded 249 locale entries for locale 'ch'
Skills Extended: Loaded 253 locale entries for locale 'en'
Skills Extended: Loaded 24 locale entries for locale 'es'
Skills Extended: Loaded 249 locale entries for locale 'fr'
Skills Extended: Loaded 27 locale entries for locale 'ge'
Skills Extended: Loaded 27 locale entries for locale 'kr'
Skills Extended: Loaded 24 locale entries for locale 'po'
Skills Extended: Loaded 174 locale entries for locale 'ru'
Skills Extended: Loaded 11 images
SpecialSlots config looks correct. Continuing to load mod...
jbs4bmx-SpecialSlots v390.0.2: Cached Successfully

    ██╗   ██╗██╗██████╗ ███████╗██████╗
    ██║   ██║██║██╔══██╗██╔════╝██╔══██╗
    ██║   ██║██║██████╔╝█████╗  ██████╔╝
    ╚██╗ ██╔╝██║██╔═══╝ ██╔══╝  ██╔══██╗
     ╚████╔╝ ██║██║     ███████╗██║  ██║
      ╚═══╝  ╚═╝╚═╝     ╚══════╝╚═╝  ╚═╝
                  Loaded
    Report Issues To @RheddElBozo on discord
[WTT-Little Drummer Boy] Database: Loaded 13 custom items.
[WTT-Little Drummer Boy] Database: Loading complete.
[Joey's ZTH Mod] Joey's ZTH++ Loaded successfully
AllTheLoot v1.0.6 initialized
Global blacklisting removed 88 items from initially 3657 handbook items
AllTheLoot assigned 3569 items to 36 available loot containers
PAutoSell: Post Aki Load Setup Complete
Started webserver at http://127.0.0.1:6969
Started websocket at ws://127.0.0.1:6969
Server is running, do not close while playing SPT, Happy playing!!
```

**Comment by SGZ-Creations** [comment]
[server-2024-08-28.log](https://github.com/user-attachments/files/16784083/server-2024-08-28.log)

**Comment by refringe** [comment]
Resolved in v1.7.3
